src/utils/coordinates-utils.js

```diff
@@ -31,18 +31,18 @@ function decimalToDMS (decimal, latitude = true, degrees = 0, minutes = 0, secon
 
 function distanceBetweenPoints (depLat, depLng, arrLat, arrLng) {
   const R = 6371e3
-  const lat1 = depLat.toRadians()
-  const lat2 = arrLat.toRadians()
-  const dlat = (arrLat - depLat).toRadians()
-  const dlng = (arrLng - depLng).toRadians()
+  const lat1 = toRadians(depLat)
+  const lat2 = toRadians(arrLat)
+  const dlat = toRadians(arrLat - depLat)
+  const dlng = toRadians(arrLng - depLng)
 
   const a = Math.sin(dlat / 2) * Math.sin(dlat / 2) +
     Math.cos(lat1) * Math.cos(lat2) *
     Math.sin(dlng / 2) * Math.sin(dlng / 2)
 
   const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a))
 
-  return R * c
+  return (R * c / 1852).toFixed(0)
```

Comment:
I don't know this math here. Why are we doing `/ 1852`? Just curious. :)

Reply:
to get Nautical miles from meters :)

Reply:
1 nm = 1852 m. Good to know. 😃